src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -0,0 +1,144 @@
+import block from 'bem-cn';
+import React, { useCallback } from 'react';
+import { Title } from '../../components/Title';
+import './profilePage.css';
+import { PageContainer } from '../../components/PageContainer';
+import { Button } from '../../components/Button';
+import { formScheme, InputNames } from '../../consts/formScheme';
+import { Form } from '../../components/Form/Form';
+import { InputFieldProfile } from '../../components/InputFieldProfile';
+import { InputFieldAvatar } from '../../components/InputFieldAvatar';
+
+const b = block('profile-page');
+
+export const ProfilePage = () => {
+  let isEditMode = true;
```

Comment:
У меня в консоли тут ошибка, что 
`react-dom.development.js:67 Warning: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.`

Нужно, наверное, в стейте хранить это значение? 
`  const [isEditMode, setIsEditMode] = useState(false);`